chatbotcore/database.py

```diff
@@ -31,7 +31,10 @@ class QdrantDatabase:
 
     def __post_init__(self):
         """Initialize database client"""
-        self.db_client = QdrantClient(host=self.host, port=self.port)
+        try:
+            self.db_client = QdrantClient(host=self.host, port=self.port)
+        except Exception as e:
+            logger.error(f"Failed to connect at {self.host}:{self.port}. Error: {e}")
```

Comment:
Let's use this `exec_info=True` instead of passing the data. As, It will be easy for tagging and scoping further. 

```suggestion
            logger.error(f"Database Connection failed", exec_info=True)
```